pypesto/visualize/optimization_stats.py

```diff
@@ -375,7 +375,7 @@ def stats_lowlevel(
         The plot axes.
     """
     fvals = result.optimize_result.fval
-    values = [res[property_name] for res in result.optimize_result.list]
+    values = [[res[property_name]] for res in result.optimize_result.list]
```

Comment:
Why is making that specifically a list within a list required?

Reply:
This is making the use of `delete_nan_inf` consistent. 
The values are given over to `delete_nan_inf` in place of the optimization result parameters. These are parameter vectors within a list, i.e. one list (or `None`) per start.